README.md

```diff
@@ -12,22 +12,49 @@ This repository contains the source for the Sunseeker Solar Car Project's main t
 
 Once you have cloned the repository onto your machine, you will want to do the following to get the telemetry up and running.
 
-First, install all of the dependencies:
+
+For Java
```

Comment:
Maybe specify the OS for these and other instructions specific to a particular OS/distribution